src/scrabble.js

```diff
@@ -1,5 +1,49 @@
-function scrabble() {
-
+function scrabble(string) {
+  if (string === null) {
+    return 0
+  } else {
+    const word = string.toUpperCase()
+    let value = 0
+    for (let i = 0; i <= word.length; i++) {
+      if (
+        word[i] === 'A' ||
+        word[i] === 'E' ||
+        word[i] === 'I' ||
+        word[i] === 'O' ||
+        word[i] === 'U' ||
+        word[i] === 'L' ||
+        word[i] === 'N' ||
+        word[i] === 'R' ||
+        word[i] === 'S' ||
+        word[i] === 'T'
+      ) {
+        value += 1
+      } else if (word[i] === 'D' || word[i] === 'G') {
+        value += 2
+      } else if (
+        word[i] === 'B' ||
+        word[i] === 'C' ||
+        word[i] === 'M' ||
+        word[i] === 'P'
+      ) {
+        value += 3
+      } else if (
+        word[i] === 'F' ||
```

Comment:
much neater and nice when u added them up to see what is being returned and what code is executed after an if statement